.ci/load/Jenkinsfile

```diff
@@ -25,10 +25,11 @@ pipeline {
+        string(name: "jvm_options", defaultValue: "", description: "Extra JVM options")
         string(name: "concurrent_requests", defaultValue: "100", description: "The number of concurrent requests to test with")
-        string(name: "duration", defaultValue: "10m", description: "Test duration. Specify with unit. Ex: 30m, 1h")
+        string(name: "duration", defaultValue: "10", description: "Test duration in minutes. Max: 280")
```

Comment:
Ah. We need to change this in `gen_params.py` to be `string(name: "duration", defaultValue: "10m", description: "Test duration. Specify with unit. Ex: 30m, 1h")` and then regenerate this snippet. Otherwise, we will break the option that's passed to Locust, which requires the unit to be passed in.

Let me know if you'd like to me to just make a PR against yours for this change for this. I'm happy to. :)

Reply:
fixed in the last commit.